website/cypress/e2e/auth/signin.cy.ts

```diff
@@ -27,13 +27,6 @@ describe("signin flow", () => {
         });
       });
   });
-  it("shows the logged in users email address if logged in with email", () => {
-    const emailAddress = "user@example.com";
-    cy.signInWithEmail(emailAddress);
-    // The user will only see the email address if the window is wide enough, not technically required as even when hidden this will find it in the page.
-    cy.viewport(1920, 1000);
-    cy.contains('[data-cy="username"]', emailAddress);
```

Comment:
Instead of deleting this test case, should we check that the `username` element exists?

Reply:
That's done in the normal test on line 26.  With that, i figured this test doesn't do much that's useful.